Cargo.toml

```diff
@@ -45,7 +45,7 @@ members = [
 ]
 
 [workspace.package]
-version = "0.2.0-alpha"
+version = "0.1.15"
```

Comment:
Please keep it as `0.2.0-alpha`. The reason is that it implies `main` branch is 1) the next version (`0.2.0`) and 2) still under development (`-alpha`)

Reply:
Example: https://github.com/tikv/tikv/blob/master/Cargo.toml#L3